Explorer/Assets/DCL/InWorldCamera/InWorldCamera/Systems/MoveInWorldCameraSystem.cs

```diff
@@ -44,7 +47,7 @@ protected override void Update(float t)
 
                 Translate(followTarget, input, t);
 
-                if (!input.MouseIsDragging)
+                if (cursor.IsLocked() || input.MouseIsDragging)
```

Comment:
I don't understand this change, so I ask just in case: why was it negating MouseIsDragging before, and now it is not? Wasn't mouse dragging allowed before and now it supports both things (locking with right-button and draging with left-button)?

Reply:
exactly. In first iteration I miss-understood requirements on how mouse lock should work in this mode